ts/pnpm-lock.yaml

```diff
@@ -282,7 +285,7 @@ importers:
         version: 18.18.7
       copy-webpack-plugin:
         specifier: ^12.0.1
-        version: 12.0.2(webpack@5.94.0(webpack-cli@5.1.4))
+        version: 12.0.2(webpack@5.94.0)
```

Comment:
@curtisman, @umeshma: Do you understand what these changes are about? I have no idea what pnpm-lock.yaml does. This type of changes is all over the file.

Reply:
But I will assume it is somehow correct and I'll still merge this if all tests pass.

Reply:
They passed. Merging.